quantum/encoder/tests/encoder_tests_split_left_eq_right.cpp

```diff
@@ -117,7 +117,7 @@ TEST_F(EncoderSplitTestLeftEqRight, TestOneClockwiseRightSent) {
     encoder_state_raw(slave_state);
 
     EXPECT_EQ(slave_state[0], 0);
-    EXPECT_EQ(slave_state[1], 0xFF);
+    EXPECT_EQ(slave_state[3], 0xFF);
```

Comment:
These tests were failing as now we're copying All of `encoder_value` over instead of only the opposite hand.